azure/flow-runner-function/src/main/java/com/marklogic/dhf/azure/FunctionDhfFlow.java

```diff
@@ -18,9 +18,9 @@
  */
 public class FunctionDhfFlow {
     /**
-     * This function listens at endpoint "/api/HttpExample". Two ways to invoke it using "curl" command in bash:
-     * 1. curl -d "HTTP Body" {your host}/api/HttpExample
-     * 2. curl "{your host}/api/HttpExample?name=HTTP%20Query"
+     * This function listens at endpoint "/api/HttpExampleDhfFlow". Two ways to invoke it using "curl" command in bash:
```

Comment:
Same thing, can you do DataHubFlow? Or - just rename this in the next JIRA ticket. Either approach is fine.

Reply:
Sure. Done.